hatch_build.py

```diff
@@ -0,0 +1,37 @@
+from logging import shutdown
+from pathlib import Path
+import os
+import shlex
+from subprocess import call
+import sys
+from hatchling.builders.hooks.plugin.interface import BuildHookInterface
+
+
+class LarkStandAloneBuildHook(BuildHookInterface):
+    PLUGIN_NAME = "lark_standalone"
+
+    def initialize(self, version, build_data):
+        L = get_logger()
+        WIN = os.name == "nt"
+        L.info("converting json grammar to python")
+        python_parser = Path("src/importnb/_json_parser.py")
+        if not python_parser.exists():
+            with python_parser.open("w") as file:
+                call(
+                    sys.executable, 
+                    *shlex.split(
+                        "-m lark.tools.standalone --propagate_positions src/nb.g",
+                        posix=~WIN,
+                    ),
```

Comment:
```suggestion
                    [
                        sys.executable,
                        *shlex.split(
                            "-m lark.tools.standalone --propagate_positions src/nb.g",
                            posix=~WIN,
                        ),
                    ],
```

Reply:
thanks for the suggestion. i sorted it out in another commit.